### PR TITLE
RAC: `<Table>` as react aria component

### PR DESCRIPTION
**Pull request by sarahgm**

### CODE REVIEW

File: packages/components/src/Table/Table.stories.tsx [hunk 1]
```diff
@@ -324,15 +244,142 @@ export const Sorting: Story = {
...
+            )}
+          </Table.Body>
+        </Table>
+        <div>Selected rows: {selected.join(', ')}</div>
+      </Stack>
+    );
+  },
+};
+
+// https://react-spectrum.adobe.com/react-aria/Table.html#nested-columns
```
[the remark this answers is not in the document]
Comment (sebald) [reply packages/components/src/Table/Table.stories.tsx:329]: Is this the error?

`TypeError: Cannot read properties of undefined (reading 'key')`

[the remark this answers is not in the document]
Comment (sarahgm) [reply packages/components/src/Table/Table.stories.tsx:329]: Yes but why... I have tried everywhere to set id and key :/

[the remark this answers is not in the document]
Comment (sarahgm) [reply packages/components/src/Table/Table.stories.tsx:329]: I have it like in the example

[the remark this answers is not in the document]
Comment (sebald) [reply packages/components/src/Table/Table.stories.tsx:331]: Might be a bug in RAC: https://github.com/adobe/react-spectrum/issues/5281

**Review by sebald — CHANGES_REQUESTED** [review]

File: docs/content/components/collection/table/table-async.demo.tsx [hunk 2]
```diff
@@ -41,23 +41,26 @@ export default () => {
...
       <Table.Header>
-        <Table.Column key="name" allowsSorting>
+        <Table.Column id="name" allowsSorting isRowHeader>
           Name
         </Table.Column>
-        <Table.Column key="height" allowsSorting>
+        <Table.Column id="height" allowsSorting>
           Height
         </Table.Column>
-        <Table.Column key="mass" allowsSorting>
```
[the remark this answers is not in the document]
Comment (sarahgm) [reply docs/content/components/collection/table/table-async.demo.tsx:50]: What do you mean? @sebald

[the remark this answers is not in the document]
Comment (sarahgm) [reply docs/content/components/collection/table/table-async.demo.tsx:50]: done

**Review by sebald — APPROVED** [review]

**Pull request merged**